frontend/test/__support__/cypress.js

```diff
@@ -130,6 +134,24 @@ export function visitAlias(alias) {
   });
 }
 
+export function createBasicAlert(variable) {
+  cy.get(".Icon-bell").click();
+  if (variable === "first alert") {
```

Comment:
Rather than passing a string to compare, I think it'd be cleaner to pass booleans. e.g.
```js
function createBasicAlert({firstAlert}={}) {
if (firstAlert) {

...

createBasicAlert({firstAlert:true})
```

Reply:
Great suggestion!